src/host/srvinit.cpp

```diff
@@ -5,30 +5,22 @@
 
 #include "srvinit.h"
 
+#include "ApiRoutines.h"
 #include "dbcs.h"
 #include "handle.h"
 #include "registry.hpp"
+#include "renderData.hpp"
 #include "renderFontDefaults.hpp"
-
-#include "ApiRoutines.h"
-
-#include "../types/inc/GlyphWidth.hpp"
-
-#include "../server/DeviceHandle.h"
-#include "../server/Entrypoints.h"
-#include "../server/IoSorter.h"
-
-#include "../interactivity/inc/ISystemConfigurationProvider.hpp"
-#include "../interactivity/inc/ServiceLocator.hpp"
+#include "../inc/conint.h"

```

Comment:
this sort seems unrelated

Reply:
actually the changes to this file seem unrelated

Reply:
Ah woops, I originally changed the signature of `SetGlyphWidthFallback`, but then reverted it to make the PR smaller.